tools/SetupFlow/DevHome.SetupFlow/Models/TaskInformation.cs

```diff
@@ -19,12 +19,33 @@ public ISetupTask TaskToExecute
         get; set;
     }
 
+    /// <summary>
+    /// The message to display in the loading screen.
+    /// </summary>
     [ObservableProperty]
     private string _messageToShow;
 
+    /// <summary>
+    /// If the status icon, green, yellow, or red, should be shown.
+    /// </summary>
     [ObservableProperty]
     private bool _statusIconGridVisibility;
 
+    /// <summary>
+    /// If the progress ring should be shown.  Only show a progress ring when the task is running.
+    /// </summary>
+    [ObservableProperty]
+    private bool _shouldShowProgressRing;

```

Comment:
The view model is a better candidate for storing this information, but since there is no VM for this model, can you please create an issue to refactor this model.

Reply:
https://github.com/microsoft/devhome/issues/1189